aws/application/application.template

```diff
@@ -894,7 +900,9 @@ Resources:
       GenerateSecret: true
       SupportedIdentityProviders: [ COGNITO ]
       AllowedOAuthFlows: [ client_credentials ]
-      AllowedOAuthScopes: [ !Sub '${pAppName}/${pApiScope1}' ]
+      AllowedOAuthScopes:
+        - !Sub '${pAppName}/${pApiScopeDcesStandard}'
+        - !If [ cNonProd, !Sub '${pAppName}/${pApiScope1}', !Ref AWS::NoValue ]
```

Comment:
We need to add a new scope for CognitoUserPoolClientDcesDrcReportService service as well same as CognitoUserPoolClientDcesReportService.

Do we need to add new scope to CognitoUserPoolResourceServer1, please could you verify :-)

Reply:
Ah yes good spot!